src/js/components/Form/stories/typescript/TypedForm.tsx

```diff
@@ -75,14 +86,23 @@ export const TypedForm = () => {
           />
           <FormField
             label="Size"
+            aria-label="size"
```

Comment:
Can we still add `htmlFor` and `id` in addition to `aria-label`?

Reply:
```suggestion
            htmlFor="size"
            id="size"
            aria-label="size"
```
I tested these changes locally and no issues are created by adding the htmlFor and id.